src/nupic/regions/sp_region.py

```diff
@@ -478,10 +465,10 @@ def _allocateSpatialFDR(self, rfInput):
 
   def compute(self, inputs, outputs):
     """
-    Run one iteration of SPRegion's compute, profiling it if requested.
+    Run one iteration, profiling it if requested.
 
-    The guts of the compute are contained in the _compute() call so that
-    we can profile it if requested.
+    :param inputs: TODO: I need help with this. -- Matt 
+    :param outputs: TODO: I need help with this. -- Matt 
```

Comment:
@scottpurdy Can you or @mrcslws provide descriptions of theses params? The expected data structures need to be defined somewhere.